.github/workflows/run-network-tests.yml

```diff
@@ -103,7 +103,7 @@ jobs:
       - name: Ensure tests are runnable
         run: yarn workspace network-tests build
       - name: Execute network tests
-        run: RUNTIME=babylon tests/network-tests/run-tests.sh full
+        run: RUNTIME=antioch tests/network-tests/run-tests.sh full
```

Comment:
If you update from antioch branch you will find this job has actually been disabled. It is for testing runtime upgrade. So `RUNTIME=name` is the starting runtime which will be upgraded. So this change isn't really necessary.

Reply:
There is no harm in keeping this change, it will be required for the next runtime upgrade integration tests for sumer

Reply:
I just copied from the last PR, where you corrected this...

If there are any benefit in reverting, I'll do that?